flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -1068,8 +1067,8 @@ public static void teardown() throws Exception {
 
 	}
 
-	public static boolean isOnTravis() {
-		return System.getenv("TRAVIS") != null && System.getenv("TRAVIS").equals("true");
+	public static boolean isOnCI() {
+		return System.getenv("IS_CI") != null && System.getenv("IS_CI").equals("true");
```

Comment:
hmm, this reminds me, we aren't caching downloads for the e2e tests on azure, are we?

Reply:
Nope, we are not. What needs to be done for that?

Reply:
Adjust the TravisDownloadCache(Factory) to detect azure environment variables and determine the correct cache directory.

Reply:
Cool, thx. Will do.